docs/FAQ.md

```diff
@@ -56,7 +56,7 @@ No
 
 ### What is the policy on supporting end-of-life(EOL) Ruby versions?
 
-We will target the latest EOL Ruby version, please check the link below the see the current EOL version.
+We will target all non EOL Ruby versions, please check the link below the see the current EOL version.
```

Comment:
Good catch, here is my stance:
- support `!eol` versions
- best effort attempt on latest EOL

If the ruby community is no longer supporting it from a security stance I don't want to encourage people to run stuff that will receive security updates. Just trying to avoid the windows windows XP equivalent here. We use gems and they are versioned for a reason, as long as we stick to semver they can run the same ancient version from 10 years ago just don't complain if you can't get new features on an old platform.

While I think that we should create our own rules independent of what sensu inc decides to support I would like to ideally align with sensu-core if possible though I can't seem to find any docs detailing this. @portertech can you either hunt down some doc or give me your thoughts?